app/scienceapi/static/admin/custom-event-admin.css

```diff
@@ -0,0 +1,17 @@
+/* Removes Timezone warning from admin */
+.timezonewarning {
+	display: none;
+}
+
+/* Remove relative datetime pickers */
+.datetimeshortcuts > * {
```

Comment:
but it will expose relative times like now. what about adding a help text about time format?